src/submission-handlers/base-handlers.ts

```diff
@@ -244,3 +244,66 @@ const multiSelectObsHandler = (field: OHRIFormField, values: Array<string>, cont
 export function teardownBaseHandlerUtils() {
   assignedObsIds = [];
 }
+/**
+ * Order handler
+ */
+export const OrderSubmissionHandler: SubmissionHandler = {
```

Comment:
@jabahum, I'm concerned that this file may grow to a point where it will be challenging to maintain. Could you consider introducing these handlers in their own file?